downstream/modules/automation-hub/proc-obtaining-org-collection-url.adoc

```diff
@@ -11,12 +11,12 @@ You can synchronize {CertifiedName} curated by your organization from `{Console}
 
 .Procedure
 
-. Log in to `{Console}` as an organization administrator.
-. Navigate to menu:Automation Hub[Repo Management].
-. Locate the *Sync URL* and click the btn:[Copy to clipboard] icon (image:copy.png[Copy,15,20]). 
-Paste the *Sync URL* in a file to use when configuring the *rh-certified* remote.
-. Click the btn:[More actions] icon image:ellipsis.png[More,5,12] and click btn:[Get token].
-. On the *Token management* page, click btn:[Load token].
+. Log in to menu:Automation Hub[Repositories].
```

Comment:
@ianf77 Based on @awcrosby comments, I believe these steps should actually be:
1. Log in to `{Console}` as an organization administrator.
2. Navigate to menu:Automation Hub[Connect to Hub].
3. Under Offline token, click btn:[Load token].
4. Click btn:[Copy to clipboard] to copy the API token.
5. Paste the API token into a file and store in a secure location.